frontend/cypress.json

```diff
@@ -1,6 +1,6 @@
 {
 	"integrationFolder": "cypress/integration",
-	"supportFile": "cypress/support/index.ts",
+	"supportFile": "cypress/support/commands.ts",
```

Comment:
This is not the way described in the docs. Keep the index.ts file and import command.ts in that file.